AZ3166/src/libraries/AzureIoT/examples/RemoteMonitoringv2/.bin/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sketch": "../RemoteMonitoring.ino",
```

Comment:
The name of ino should be the same of the folder name, so could you use RemoteMonitoringV2.ino?